### DISCUSSION TITLE
Deepin V23 安装问题

### DISCUSSION THREAD

**Discussion by wolono** [1111mp/nvm-desktop#120] [Q&A]
安装后运行软件卡在界面
![image](https://github.com/user-attachments/assets/6489e33a-0fe7-4f81-9c03-c29ecb65c3c1)
所有菜单都无法点

**Comment by 1111mp** [comment]
你可以看看 deepin v23 的基于哪个版本的debian系统 目前来说没有计划支持兼容 deepin 系统 你也可以根据文档自己编译成 `AppImage` 格式的包
